### PR TITLE
git describe != semver, so look in hack/print-workspace-status.sh

### PR DESCRIPTION
**Pull request by ixdy**
/assign @krzyzacy @zmerlynn 

x-ref #4315

### CODE REVIEW

File: scenarios/kubernetes_kops_aws.py [hunk 1]
```diff
@@ -212,7 +212,8 @@ def main(args):
         cmd.extend(['-v', '/var/run/docker.sock:/var/run/docker.sock'])
 
     if args.stage is not None:
-        version = check_output('git', 'describe', '--always').strip()
+        version = check_output('bash', '-c', 'hack/print-workspace-status.sh '
+                               '| grep gitVersion | awk \'{print $2}\'').strip()
```
Comment (ixdy) [inline scenarios/kubernetes_kops_aws.py]: I could write a python loop to do this instead, but

**Comment by krzyzacy** [comment]
/lgtm
(?)
/shrug

**Comment by ixdy** [comment]
ok, I made this more pythonic. it has slightly better error checking now.

**Comment by zmerlynn** [comment]
/lgtm
still

**Pull request merged**